### PR TITLE
Feature/quicklinks v2

### PR DESCRIPTION
**Pull request by fatimaaurangzeb0640**
# Description

Implements quicklinks using V2 endpoints by updating the necessary parameters for the button and checkout required in the V2 flow.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] UI change (Front-end UI update without breaking changes or business logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Description

1. **What is the current behavior?** (You can also link to an open issue here)

2. **What is the new behavior (if this is a feature change)?**

3. **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/zoid/checkout/checkout.jsx [hunk 1]
```diff
@@ -52,42 +51,33 @@ export const Checkout = create({
     ).render(dom({ doc }));
   },
 
+
```
Comment (rashid007x) [inline src/zoid/checkout/checkout.jsx:54]: you didn't include customer token prop inside props object, it's required to open customer flow if tracker doesn't include it.

**Comment by ziyadparekh** [comment]
@fatimaaurangzeb0640 PR description is empty - please add details

File: src/configs/domains.js [hunk 2]
```diff
@@ -1,94 +1,100 @@
...
 export const Config = {
+  env: ENV.LOCAL,
 
-	env: ENV.LOCAL,
+  // checkoutUris: {
+  // 	[ ENV.LOCAL ]: "/pay",
+  // 	[ ENV.DEV ]: "/checkout/pay",
+  // 	[ ENV.SANDBOX ]: "/checkout/pay",
+  // 	[ ENV.PRODUCTION ]: "/checkout/pay"
+  // },
```
Comment (ziyadparekh) [inline src/configs/domains.js]: why are we leaving these comments here?

File: src/zoid/checkout/checkout.jsx (hunk 1 above)
Comment (ziyadparekh) [reply src/zoid/checkout/checkout.jsx:54]: I don't think we are supporting the `customer` flow through this for now - just payment flow for guests and users.

we should think about how to support address token though

Comment (fatimaaurangzeb0640) [reply src/zoid/checkout/checkout.jsx:54]: will make a ticket for this

**Review by ziyadparekh — APPROVED** [review]

**Pull request merged**